akalisten/templates/lotsude/index.j2

```diff
@@ -26,30 +26,32 @@
             </button>
             <div class="accordion-content mucke" id="mucke-{{ poll.id }}" hidden>
-                {% if poll.url %}
-                    <div class="entry-link">
-                        🔗 <a href="{{ poll.url }}" target="_blank" rel="noopener noreferrer">Hier zur Mucke eintragen</a>
-                    </div>
-                {%  endif %}
-                {% if poll.mucken_info.has_any_info or poll.mucken_info.additional %}
-                    <div class="accordion" data-state="closed">
-                        <button class="accordion-header" aria-expanded="false" aria-controls="mucke-{{ poll.id }}-infos">
-                            Mucken-Infos
-                            <span class="accordion-icon">⬇️</span>
-                        </button>
-                        <div class="accordion-content mucken-infos" id="mucke-{{ poll.id }}-infos" hidden>
-                            {% if poll.mucken_info.has_any_info %}
-                                {{ poll.mucken_info.html_info }}
-                            {% endif %}
-                            {% if poll.mucken_info.additional %}
-                                <p>
-                                    <b>📝 Zusätzliche Informationen:</b>
-                                    {{ poll.mucken_info.html_additional }}
-                                </p>
-                            {% endif %}
+                <div class="mucken-header" id="mucke-{{ poll.id }}-header">
+                    {% if poll.mucken_info.has_any_info or poll.mucken_info.additional %}
+                        <div class="accordion" data-state="closed">
+                            <button class="accordion-header" aria-expanded="false" aria-controls="mucke-{{ poll.id }}-infos">
+                                Mucken-Infos
+                                <span class="accordion-icon">⬇️</span>
+                            </button>
+                            <div class="accordion-content mucken-infos" id="mucke-{{ poll.id }}-infos" hidden>
+                                {% if poll.mucken_info.has_any_info %}
+                                    {{ poll.mucken_info.html_info }}
+                                {% endif %}
+                                {% if poll.mucken_info.additional %}
+                                    <p>
+                                        <b>📝 Zusätzliche Informationen:</b>
+                                        {{ poll.mucken_info.html_additional }}
+                                    </p>
+                                {% endif %}
+                            </div>
                         </div>
-                    </div>
-                {% endif %}
+                    {% endif %}
+                    {% if poll.url %}
+                        <button class="entry-btn">
+                            🔗 <a href="{{ poll.url }}" target="_blank" rel="noopener noreferrer">Hier zur Mucke eintragen</a>
+                        </button>
```

Comment:
I don't know why. But the button creates a page overflow on mobile screens.
In addition, mixing button and link could lead to problems, as described here: https://stackoverflow.com/a/59908341
As this is only visually a button, it should be sufficient to set the corresponding styles.